src/bridge/cockpitdbusjson.c

```diff
@@ -2703,7 +2703,13 @@ on_name_vanished (GDBusConnection *connection,
 
   CockpitDBusPeer *peer = g_hash_table_lookup (self->peers, name);
   if (peer)
-    cockpit_dbus_cache_set_name_owner (peer->cache, NULL);
+    {
+      // Using the empty string as the name owner means that the cache
```

Comment:
I was going to complain about C++ style comments, but we already have a few in our code.